### PR TITLE
Fix heroku deployment

### PR DESCRIPTION
**Pull request by mg2239**
### Summary 

This pull request fixes Heroku's deployment.
### Breaking Changes  

<!-- - Database schema change (anything that changes DynamoDB document structure)
<!-- - Other change that could cause problems (Detailed in notes)

### CODE REVIEW

File: frontend/src/components/ExportPreview/ExportPreview.tsx [hunk 1]
```diff
@@ -43,19 +43,19 @@ const ExportPreview = () => {
...
-         data={downloadData}
-         filename={`scheduledRides_${today}.csv`}
-         className='hidden'
-         ref={csvLink}
-         target='_blank'
-      />
+          data={downloadData}
+          filename={`scheduledRides_${today}.csv`}
+          className='hidden'
+          ref={csvLink}
```
Comment (lsizemore8) [inline frontend/src/components/ExportPreview/ExportPreview.tsx:56]: Did anything actually change here or was this a weird formatting change?

File: frontend/src/components/ExportPreview/ExportPreview.tsx [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 import styles from './exportPreview.module.css';
 import { useReq } from '../../context/req';
 import ExportButton from '../ExportButton/ExportButton';
 import { useDate } from '../../context/date';
-import { CSVLink } from "react-csv";
 
 const ExportPreview = () => {
   const [drivers, setDrivers] = useState<Driver[]>([]);
-  const [downloadData, setDownloadData] = useState<string>("");
+  const [downloadData, setDownloadData] = useState<string>('');
```
Comment (lsizemore8) [inline frontend/src/components/ExportPreview/ExportPreview.tsx:13]: Is there any reason to use single quotes vs double?

**Review by lsizemore8 — APPROVED** [review]
Looks good, just left a couple questions. This was a quick fix so obviously not a big concern but the formatting changes were a tad strange

File: frontend/src/components/ExportPreview/ExportPreview.tsx (hunk 2 above)
Comment (mg2239) [reply frontend/src/components/ExportPreview/ExportPreview.tsx:13]: single quotes is whats part of our eslint rule, it just auto corrected on save

**Pull request closed without merging**